index.js

```diff
@@ -94,6 +98,9 @@ const rxhr = options => {
         request.onload = onReqLoad
         request.onerror = onReqError
         request.ontimeout = onReqTimeout
+        if (options.progressObserver && options.progressObserver.next) {
```

Comment:
Not sure `progressObserver[$$observable]` compatible with other libraries
Rxjs is fine